bdanmaku.lua

```diff
@@ -2,11 +2,12 @@
 -- License: MIT
 -- Homepage: https://github.com/UlyssesZh/bdanmaku
 
+table.unpack = table.unpack or unpack -- 5.1 compatibility
 local CURL = mp.get_opt('curl_executable') or 'curl'
 local BILIASS = mp.get_opt('biliass_executable') or 'biliass'
-local TMPDIR = mp.get_opt('tmpdir') or '/tmp'
+local TMPDIR = mp.get_opt('tmpdir') or '/tmp/danmaku/'
 local BILIASS_OPTS = {}
-for token in (mp.get_opt('biliass_options') or ''):gmatch('[^%s]+') do
+for token in (mp.get_opt('biliass_options') or '-fs 48 -a 0.5 -p 270 -dm 10'):gmatch('[^%s]+') do
```

Comment:
Use unabbreviated option names so that people don't have to look at the help page to know what is done here.

Reply:
Update in [1940e95](https://github.com/UlyssesZh/bdanmaku/pull/1/commits/1940e9522a046f05fec04b2e7dc04c2f0f97841a).